### ISSUE TITLE
Upgrading to current tokio

### ISSUE THREAD

**Issue by 0xpr03** [jonhoo/async-ssh#5]
Hi,
are there any plans for upgrading this from tokio-core to the current tokio implementation?

**Comment by jonhoo** [comment]
I'm hoping to get back to this at some future as part of my [live-coding streams](https://www.youtube.com/c/JonGjengset), which is where [we built](https://www.youtube.com/watch?v=RBQwZthJjoM) what's here already. You can vote for that particular topic [here](https://jon.thesquareplanet.com/live-coding/), but it doesn't look like it's about to come up any time very soon. In the meantime I'd be happy to review a PR, or you could try to use thrussh directly :)

**Comment by 0xpr03** [comment]
Hey thanks for the answer. I'll try using trussh directly for my mockup. Is there a big downside besides syntactic sugar ?

**Comment by jonhoo** [comment]
No, I think that's mostly it. I haven't tried the latest release on thrussh, which apparently fixes a lot of the ergonomic issues, so I think it'll mostly just require somewhat more verbose code :)
